zeeguu_api/api/teacher_dashboard.py

```diff
@@ -432,3 +436,81 @@ def update_cohort(cohort_id):
     except sqlalchemy.exc.IntegrityError:
         flask.abort(400)
         return 'IntegrityError'
+
+
+@api.route("/upload_files/<cohort_id>", methods=["POST"])
+@with_session
+def upload_file(cohort_id):
+    '''
+        uploads a file for a cohort with input from a POST request
+    '''
+    if (not has_permission_for_cohort(cohort_id)):
+        flask.abort(401)
+    try:
+        for file in json.loads(request.data):
```

Comment:
file used to be a builtin... also, technically you're not getting a file but rather an `article_data` it seems to me